src/java/org/apache/cassandra/repair/autorepair/AutoRepairStateFactory.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.repair.autorepair;
+
+import org.apache.cassandra.repair.autorepair.AutoRepairConfig.RepairType;
+
+public class AutoRepairStateFactory
+{
+    public static AutoRepairState getAutoRepairState(RepairType repairType)
```

Comment:
Small preference:  It could be better to just put this on `RepairType`, can just define a `AutoRepairState getAutoRepairState()` on the `RepairType` enum.

Reply:
Done